runtime/sema/type.go

```diff
@@ -4704,7 +4745,7 @@ func (*ReferenceType) IsEquatable() bool {
 	return true
 }
 
-func (*ReferenceType) TypeAnnotationState() TypeAnnotationState {
+func (t *ReferenceType) TypeAnnotationState() TypeAnnotationState {
```

Comment:
The receiver variable isn't used in the body, maybe revert:

```suggestion
func (*ReferenceType) TypeAnnotationState() TypeAnnotationState {
```